src/cljs/athens/views/node_page.cljs

```diff
@@ -527,8 +521,15 @@
                                                        :menu/y    (.. rect -bottom)}))))
                    :style    page-menu-toggle-style}
            [:> mui-icons/MoreHoriz]]
-          (:title/local @state)]
-          ;;(parse-renderer/parse-and-render title uid)]
+          (when-not timeline-page?
+            [autosize/textarea
+             {:value         (:title/local @state)
+              :id            (str "editable-uid-" uid)
+              :class         (when (= editing-uid uid) "is-editing")
+              :on-blur       (fn [_] (handle-blur node state linked-refs))
+              :on-key-down   (fn [e] (handle-key-down e uid state children))
+              :on-change     (fn [e] (handle-change e state))}])
+          [parse-renderer/parse-and-render (:title/local @state) uid]]
```

Comment:
Why does the location of this snippet need to change?

Reply:
The h1 and textarea input should be adjacent to each other that's why I moved this down. I'm not entirely sure why that's the case but I'm guessing it's the z-index on css.